pages/api/extension/trigger.ts

```diff
@@ -48,12 +49,19 @@ async function triggerDPU(url: string, userEmail: string) {
 		console.error(`[triggerDPU] Unable to get user for alias ${userEmail}, error = ${err}`);
 		throw new Error("Unable to get user from db");
 	});
-	if (users?.length == 0) {
+	if (!users || users?.length == 0) {
 		console.error(`[triggerDPU] Unable to find user for alias ${userEmail}`);
 		throw new Error("User not found in db");
 	}
+	const userId = users[0].id;
+	if (!userId) {
+		console.error(`[triggerDPU] Unable to find user id for user ${JSON.stringify(users[0])}`);
+		throw new Error("User ID not found in db user");
+	}
     // get repo config
+    const repoConfig = await getRepoConfigByUserAndRepo(repoProvider, repoName, repoOwner, userId);
```

Comment:
Low LOGAF


This will only work for individual installations (and for the admin who installs the DPU for a project installation).

Maybe we can have a more liberal condition here.

Reply:
Lets send a default repo config if we don't get anything from db. What should the defaults be?